### ISSUE TITLE
`colorcolumn` is a bit obfuscating

### ISSUE THREAD

**Issue by LuisxSullivaN** [catppuccin/nvim#104]
hey! i have set the option `colorcolumn` and the color makes the letters under it a bit harder to read. Maybe use a lower contrast color? like the same used for the option `cursor` and `cursorcolumn` or something like that
![image](https://user-images.githubusercontent.com/29529436/152711153-07a120b3-2c23-48ec-a455-cd969d076b12.png)

**Comment by timoleistner** [comment]
Its a bit confusing for me how to set custom highlight groups too.
The example from the readme only _somehow_ works.
`:h colorcolumn` led me to the hl-ColorColumn highlightgroup.
I've got the color table from the plugin like this: `:lua print(vim.inspect(require'catppuccin.api.colors'.get_colors()))`
Setting the ColorColumn highlight group to anything else changes the color to... none I guess.
```
local colors = require'catppuccin.api.colors'.get_colors()
catppuccin.remap({ ColorColumn = {fg = colors.catppuccin1 }, })
```
I've tried some variations like `colors.flamingo`, `colors.catppuccin7` or directly setting it to a hex value. None of them yielded a different result.
Help greatly appreciated :) Nice colorscheme btw.
